src/components/Banner/Banner.tsx

```diff
@@ -94,6 +99,10 @@ const StyledButton = styled(Button)`
   padding: 0;
 `;
 
+const Text = styled(BaseText)`
+  max-width: 75ch;
```

Comment:
`ch` unit is not the same for every font. In our case we probably should use `64ch` to get ~75 characters in one row.